src/khoj/interface/web/chat.html

```diff
@@ -335,7 +345,14 @@
             box-shadow: 0 0 11px #aaa;
         }
         #chat-input {
+            font-family: roboto, karma, segoe ui, sans-serif;
             font-size: medium;
+            height: 40px;
+            resize: none;
+            overflow-y: hidden;
+            max-height: 200px;
+            box-sizing: border-box;
+            padding: 10px;
```

Comment:
It'd be great if we can update the styling to have the padding, margin and line-height like before. Something like below:

```suggestion
            height: 54px;
            resize: none;
            overflow-y: hidden;
            max-height: 200px;
            box-sizing: border-box;
            padding: 15px;
            line-height: 1.5em;
            margin: 0;
```